packages/fxa-admin-panel/server/lib/server.ts

```diff
@@ -71,13 +81,52 @@ app.get('/__version__', (_, res) =>
   res.type('application/json').send(JSON.stringify(version))
 );
 
+function injectMetaContent(html: any, metaContent: any = {}) {
```

Comment:
These don't have to be `any`s.  We do know some type info on them from how they are used.  `html` has a `replace` function and `metaContent` has keys.  At the very least they should be types that satisfy those requirements.